src/pages/Question.js

```diff
@@ -1,69 +1,59 @@
 import React, {useState} from 'react'
 import Logo from '../components/Logo.js'
-import Header from '../components/Header.js'
-import  {questions} from '../config';
 import Answer from '../components/Answer.js'
 import '../styles/questions.scss';
-import PreviousButton from '../components/PreviousButton.js';
+import PreviousButton from '../assets/PreviousButton.js';
 
-const Question = () => {
-    const categories = ['food', 'sleep', 'sport', 'misc'];
-    const [currentCategory, setCurrentCategory] = useState(0)
-    const [currentQuestion, setCurrentQuestion] = useState(1)
-
-    let categoryObject = questions[categories[currentCategory]]
-    let questionObject = categoryObject.questions[currentQuestion]
+const Question = (question) => {
 
     const nextQuestion = () => {
-
-        setCurrentQuestion(currentQuestion + 1)
-        if (!((currentQuestion+1) in categoryObject.questions)) {
-            if (currentQuestion === 10) {
-                window.location.href = "/results";
-            } else {
-                setCurrentCategory(currentCategory + 1)
-            }
-        }
+        // setCurrentQuestion(currentQuestion + 1)
+        // if (!((currentQuestion+1) in categoryObject.questions)) {
+        //     if (currentQuestion === 10) {
+        //         window.location.href = "/results";
+        //     } else {
+        //         setCurrentCategory(currentCategory + 1)
+        //     }
+        // }
     }
     const previousQuestion = () => {
-        setCurrentQuestion(currentQuestion - 1)
-        if (currentQuestion === 1) {
-            window.location.href = "/";
-        }
-        else if (!((currentQuestion-1) in categoryObject.questions)) {
-            setCurrentCategory(currentCategory - 1)
-        }
+        // setCurrentQuestion(currentQuestion - 1)
+        // if (currentQuestion === 1) {
+        //     window.location.href = "/";
+        // }
+        // else if (!((currentQuestion-1) in categoryObject.questions)) {
+        //     setCurrentCategory(currentCategory - 1)
+        // }
     }
 
     const questionNumber = () => {
-        return currentQuestion < 10 ? '0' + currentQuestion : currentQuestion
+        return question.content.number < 10 ? '0' + question.content.number : question.content.number
```

Comment:
Often question.content gets put in a new variable, but for now it's fine

Reply:
I agree it makes it alot more organized and prevents making misstakes